src/features/profile/DangerCard.js

```diff
@@ -15,6 +15,10 @@ import { CardContentSection } from "../../components"
 import { selectAllEntries, deleteAllEntries } from "../entries"
 import { useSettings } from "../settings"
 
+import { FATHOM_DELETE_ALL_ENTRIES, FATHOM_DELETE_CONFIRMED } from "../../constants";
+import {trackGoal} from "../tracking"
```

Comment:
**thought:** Looks like prettier is not working properly. Should be a space before and after `trackGoal`

Reply:
Should be a space before and after trackGoal #303
b2d9f3b